linux_os/guide/system/software/integrity/crypto/harden_openssl_crypto_policy/rule.yml

```diff
@@ -0,0 +1,43 @@
+documentation_complete: true
+
+prodtype: rhel8,rhcos4
+
+title: 'Harden OpenSSL Crypto Policy'
+
+description: |-
+    Crypto Policies are means of enforcing certain cryptographic settings for
+    selected applications including OpenSSL. OPenSSL is by default configured to
+    modify its configuration based on currently configured Crypto-Policy.
+    However, in certain cases it might be needed to override the Crypto Policy
+    specific to OpenSSL r and leave rest of the Crypto Policy intact. This can
+    be done by dropping a file named <tt>opensslcnf-xxx.config</tt>, replacing
+    <tt>xxx</tt> with arbitrary identifier, into
+    <tt>/etc/crypto-policies/local.d</tt>. This has to be followed by running
+    <tt>update-crypto-policies</tt> so that changes are applied. Changes are
+    propagated into <tt>/etc/crypto-policies/back-ends/opensslcnf.config</tt>.
+    This rule checks if this file contains predefined <tt>Ciphersuites</tt>
+    variable configured with predefined value.
+
+rationale: |-
+    The Common Criteria requirements specify that certain parameters for OpenSSL
+    are configured e.g. cipher suites. Currently particular requirements
+    specified by CC are stricter compared to any existing Crypto Policy.
+
+severity: medium
+
+identifiers:
+    cce@rhel8: 84286-4
+    cce@rhcos4: 84285-6
+
+references:
```

Comment:
@redhatrises  Hello, could you please verify if these references for nist and stig are correct? Thank you.